mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

```diff
@@ -82,3 +83,61 @@ def get_invoices(self, **kwargs) -> List[Dict]:
         connection = self.handler.connect()
         invoices = paypalrestsdk.Invoice.all(kwargs, api=connection)
         return [invoice.to_dict() for invoice in invoices['invoices']]
+
+
+class OrdersTable(APITable):
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'orders',
+            self.get_columns(None)
+        )
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        # order_id = '7SP60397AN682533Y'
+        order_id = None
+
+        for condition in where_conditions:
+            if condition.column.name == 'order_id':
+               order_id = condition.value
+            break  
+
+        if order_id is None:
+            raise ValueError("order_id must be provided in the WHERE clause of the query")
+        
+        selected_columns = self.get_columns(order_id)
+        
+        orders_df = pd.json_normalize(self.get_orders(count=result_limit,order_id=order_id))
+        select_statement_executor = SELECTQueryExecutor(
+            orders_df,
+            selected_columns,
+            where_conditions,
+            order_by_conditions
+        )
+        orders_df = select_statement_executor.execute_query()
+
+        return orders_df
+    
+    def get_columns(self,order_id) -> List[Text]:
+        return pd.json_normalize(self.get_orders(order_id)).columns.tolist()
+
+    def get_orders(self, order_id, **kwargs) -> List[Dict]:
+       headers = {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer A21AAIUR5R-kPcdVNm6D6zNxxFlSFaE2smdAEZhGFAAsq92VGjrHYBjvZyMh4dzDklXjpDj3rL7vvDRmf5S4fHXcyK0PDwaVg',
```

Comment:
This should be alredy conected using the CREATE DATABASE statement. Check connection method